src/components/MyHouses.jsx

```diff
@@ -0,0 +1,105 @@
+/* eslint-disable no-alert */
+/* eslint-disable jsx-a11y/alt-text */
+/* eslint-disable react-hooks/rules-of-hooks */
+/* eslint-disable react/button-has-type */
+import React, { useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { useNavigate } from 'react-router-dom';
+import { getCurrentUser } from '../redux/user/currentUserSlice';
+import { deleteHouse } from '../redux/house/deleteHouseSlice';
+// import { updateHouse } from '../redux/house/updateHouseSlice';
+import NavBar from './navBar/NavBar';
+
+const MyHouses = () => {
+  const dispatch = useDispatch();
+  const currentUser = useSelector((state) => state.currentUser);
+  const navigate = useNavigate();
+
+  useEffect(() => {
+    dispatch(getCurrentUser());
+  }, [dispatch]);
+
+  // Check if user property exists before trying to access it
+  const user = currentUser && currentUser.currentUser;
+
+  const modal = document.getElementById('myModal');
+  const confirmButton = document.getElementById('confirm-button');
+  const cancelButton = document.getElementById('cancel-button');
+
+  const handleDeleteHouse = (houseId) => {
+    modal.style.display = 'flex';
+    confirmButton.addEventListener('click', () => {
+      dispatch(deleteHouse(houseId));
+      window.location.reload();
```

Comment:
[OPTIONAL]
Use react-router-dom's useHistory hook instead of window.location.reload()